### PR TITLE
add old code compat to the readme

### PR DESCRIPTION
**Pull request by haxscramper**
This part had already been present in the readme for some time. 

Original version was

```
<details>
<summary class="blue">Will this break my Nim code?</summary>
</br>
Maybe. Many experienced users will know that a lot of current code 'works' because of various hacks, or create hacks themselves to make code work. See <a href="https://github.com/nim-works/nimskull/issues/8">#8</a> and the <a href="https://github.com/nim-works/nimskull#direction">direction</a> for more on this.
</details>
```  

I rephrased new to an extended, "maybe, but we won't have any guarantees on that matter"

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -228,6 +228,12 @@ existing contributors are already familiar with the current code base. We chose
 to evolve it.
 </details>
 
+<details>
+<summary class="blue">Will this break my Nim code?</summary>
+</br>
+Maybe. This project ais to become a different programming language, so some breakages can be expected. At the same time there are a lot of features that we intend to keep, so it is highly likely that most code will require only superficial alterations if any.
```
Comment (saem) [inline readme.md]:
```suggestion
This project aims to become a different programming language, so do not expect source code compatibility. If you want Nim then use it.

```

Comment (saem) [reply readme.md]: Done, let's not waste time pretending any code will work.

Comment (ShalokShalom) [reply readme.md]: Much better

**Pull request merged**